solr/core/src/java/org/apache/solr/core/FileSystemConfigSetService.java

```diff
@@ -259,6 +260,14 @@ public FileVisitResult postVisitDirectory(Path dir, IOException ioException) {
     return filePaths;
   }
 
+  private String normalizePathToForwardSlash(String path) {
+    return path.replace(File.separatorChar, '/');
+  }
+
+  private String normalizePathToOSSeparator(String path) {
```

Comment:
[Google Java Style guide](https://google.github.io/styleguide/javaguide.html#s5.3-camel-case) would say "Os" end not "OS" here.

Reply:
Fixed in 836f33ad38a2bcca839e37d63ff35a78acbee404